modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -354,7 +326,6 @@ public PartitionReplicaLifecycleManager(
         this.placementDriver = placementDriver;
         this.txManager = txManager;
```

Comment:
It looks like, this field is not needed anymore. You can get it via `zoneResourcesManager`. WDYT?

Reply:
> You can get it via zoneResourcesManager

I don't like this approach, because it looks weird that `zoneResourcesManager` is responsible for having a `txManager` in it.

> Also, the possible option is to remove this field from zoneResourceManager

I don't like this approach, because we require passing an extra parameter, which is a constant and is known during creation.

What's wrong with having this field in both places? Why do you think it's bad?

Reply:
Perhaps, you are right.